db/migrate/20210518152311_move_image_uploads_to_carrierwave_path.rb

```diff
@@ -0,0 +1,31 @@
+# move any previously uploaded images not already inside 'original' version directory for Carrierwave to new location
+class MoveImageUploadsToCarrierwavePath < Mongoid::Migration
+  def self.up
+    branding_images = UserAssetService.get_local_assets(asset_type: :branding_images)
+    files_by_dir = {}
+    branding_images.each do |pathname|
+      dir_path, filename = pathname.split
+      files_by_dir[dir_path] ||= []
+      files_by_dir[dir_path] << filename
+    end
+    files_by_dir.each do |dir_path, files|
+      folder_name = File.basename(dir_path)
+      if folder_name != 'original'
+        carrierwave_dir = dir_path.join('original')
+        files.each do |file|
+          current_path = dir_path.join(file)
+          new_path = carrierwave_dir.join(file)
+          FileUtils.mkdir_p(carrierwave_dir)
```

Comment:
since you're using mkdir -p, I think you could have simplified this script by not bothering to create the files_by_dir object, and just iterating directly over files.  AFAIK, mkdir -p no-ops if the directory already exists.  Again, this is just FYI, not worth changing the code

Reply:
You are right - it does noop if it already exists.  I hadn't thought to rely on that, which was why I organized it into the hash of folder => array of files so I could just create the directory once and then move the files.  I'll keep that in my back pocket for next time.